docs/ledger/migrations/v5.0-migration.md

```diff
@@ -0,0 +1,217 @@
+
+#### Msg Service
+
+- [v1.MsgAddCreditType](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.v1#regen.ecocredit.v1.MsgAddCreditType)
+
+- [v1.MsgAddCreditTypeResponse](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.v1#regen.ecocredit.v1.MsgAddCreditTypeResponse)
+
+- [v1.MsgSetClassCreatorAllowlist](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.v1#regen.ecocredit.v1.MsgSetClassCreatorAllowlist)
+
+- [v1.MsgSetClassCreatorAllowlistResponse](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.v1#regen.ecocredit.v1.MsgSetClassCreatorAllowlistResponse)
+
+- [v1.MsgAddClassCreator](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.v1#regen.ecocredit.v1.MsgAddClassCreator)
+
+- [v1.MsgAddClassCreatorResponse](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.v1#regen.ecocredit.v1.MsgAddClassCreatorResponse)
+
+- [v1.MsgRemoveClassCreator](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.v1#regen.ecocredit.v1.MsgRemoveAllowedDenom)
+
+- [v1.MsgRemoveClassCreatorResponse](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.v1#regen.ecocredit.v1.MsgRemoveClassCreator)
+
+- [v1.MsgUpdateClassFee](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.v1#regen.ecocredit.v1.MsgUpdateClassFee)
+
+- [v1.MsgUpdateClassFeeResponse](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.v1#regen.ecocredit.v1.MsgUpdateClassFeeResponse)
+
+- [v1.MsgAddAllowedBridgeChain](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.v1#regen.ecocredit.v1.MsgAddAllowedBridgeChain)
+
+- [v1.MsgAddAllowedBridgeChainResponse](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.v1#regen.ecocredit.v1.MsgAddAllowedBridgeChainResponse)
+
+- [v1.MsgRemoveAllowedBridgeChain](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.v1#regen.ecocredit.v1.MsgRemoveAllowedBridgeChain)
+
+- [v1.MsgRemoveAllowedDenomResponse](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.v1#regen.ecocredit.v1.MsgRemoveAllowedDenomResponse)
+
+- [v1.MsgUpdateBatchMetada](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.v1#regen.ecocredit.v1.MsgUpdateBatchMetada)
+
+- [v1.MsgUpdateBatchMetadaResponse](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.v1#regen.ecocredit.v1.MsgUpdateBatchMetadaResponse)
+
+- [v1.MsgRetire](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.v1#regen.ecocredit.v1.MsgUpdateBatchMetadaResponse)
+
+  - added `reason`
+
+- [v1.MsgSend](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.v1#regen.ecocredit.v1.MsgSend)
+
+  - updated `MsgSend.SendCredits` (see types)
+
+- [v1.MsgCreateBatch](https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.v1#regen.ecocredit.v1.MsgCreateBatch)
+
+  - updated `MsgSend.MsgCreateBatch` (see types)
+
+#### Query Service
```

Comment:
Can we do the same as suggested above and list what was added first and then what was updated?